devshell.nix

```diff
@@ -0,0 +1,7 @@
+{ pkgs, ... }:
+pkgs.mkShellNoCC {
+  packages = with pkgs; [
+    mkdocs
```

Comment:
This creates a simple devshell with the relevant dependencies.

You can drop into this shell with `nix develop`, or if you have `direnv` installed you just `direnv allow` when prompted and then it will automatically load/unload when you enter the root directory.